### DISCUSSION TITLE
SKIOVOX BLOCKED BY ADMIN

### DISCUSSION THREAD

**Discussion by Mazedotexe** [3kh0/ext-remover#855] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I don't know if it's only my school or something, but now everything says "the person who set up this computer has chosen to block this site" which makes me assume that they found a URL allowlist in kiosk apps and put an empty list in it. This absolutely sucks. The last time I used my chromebook normally was a while ago, and I have a ton of stuff I've done on Skiovox that's just straight down the drain. I am devastated.

**Comment by AshtonDavies** [comment]
Have you tried other apps and methods for it? Also, what's your version? Probably admin action taking place.

**Reply by Mazedotexe** [reply]
That’s exactly what I thought. 114, my only kiosk app is SecureTestBrowser, it’s clearly the sysadmin. If you have a bunch of stuff on skiovox, probably get a cookie exporter so you can import them onto a profile on your personal computer/something like that so you don’t lose a whole-ass account like I did :|

**Comment by 3kh0** [comment]
Send your policies, I have URLBlocklist and it does not apply when in kiosk mode

**Reply by Mazedotexe** [reply]
Yes, me too, but it seems like there is some sort of policy that blocks all URLs on kiosk mode. I will DM you the json file on Discord

**Reply by leahanderson1** [reply]
You could probably use a usb stick or smthing to transport the files from kiosk mode to your user account.

**Reply by Mazedotexe** [reply]
@OtterCodes101 how tf would that work?

**Reply by leahanderson1** [reply]
open file manager, copy the files to the usb drive, login as your normal user, copy the files

**Reply by abditube1345** [reply]
no way to bypass inbulit blocker

**Comment by rwg100** [comment]
sounds like a skill issue on the admins end

**Reply by 3kh0** [reply]
I am pretty sure it is impossible to patch from a sysadmin, but I could be wrong

**Reply by AshtonDavies** [reply]
I would think the same, but I doubt it's patched. It just might not work with securetestbrowser or something.

**Reply by leahanderson1** [reply]
works with securetestbrowser for me just saying

**Reply by AshtonDavies** [reply]
Is your version the same or higher?

**Reply by leahanderson1** [reply]
my chromebook is on 114(LTS)

**Reply by AshtonDavies** [reply]
Then it's definitely an admin block.

**Comment by akabutnicer** [comment]
Eh, I knew this would eventually happen. This is as far as an admin can go to restrict Skiovox. I really could care less what happens to this.s

**Comment by Mazedotexe** [comment]
I've been using Skiovox for a while, this was not my first time. Skiovox just did this after like 2-3 months of me using. It's definitely caused by sysadmins.

**Reply by Mazedotexe** [reply]
Skiovox was amazing; my chromebook immediately unlocked after I opened it, I could freely use inspect for developer purposes, I could use extensions, and I literally never used it for bad purposes (e.g., playing games in class instead of listening), so this is very disappointing. The fact that they patched it almost makes me want to confront my (very nice) sysadmin and ask for it back. I tried to work on a project during lunch break a while ago and then I remembered that I couldn't use the Network tab in inspect because I couldn't inspect at all. And I keep forgetting that I actually have to login using the very slow login system my district uses intead of just opening it up and contuing to work. I even used Skiovox for actual schoolwork! This is so disappointing. Half of the stuff I did was on my chromebook (because my computer is broken) and now I'm kind of caught in this weird space. It's almost like withdrawal from a chrome exploit 😐

**Reply by funkymokey** [reply, marked as answer]
you can easily buy a dell chromebook from Walmart for 50 USD my school uses the same they just overprice it when you break one

**Reply by leahanderson1** [reply]
you can easily buy a slightly used chromebook for 2 dollars at {INSERT LOCAL THRIFT STORE HERE} too

**Comment by cloneyfirey** [comment]
> Skiovox was amazing; my chromebook immediately unlocked after I opened it, I could freely use inspect for developer purposes, I could use extensions, and I literally never used it for bad purposes (e.g., playing games in class instead of listening), so this is very disappointing. The fact that they patched it almost makes me want to confront my (very nice) sysadmin and ask for it back. I tried to work on a project during lunch break a while ago and then I remembered that I couldn't use the Network tab in inspect because I couldn't inspect at all. And I keep forgetting that I actually have to login using the very slow login system my district uses intead of just opening it up and contuing to work. I even used Skiovox for actual schoolwork! This is so disappointing. Half of the stuff I did was on my chromebook (because my computer is broken) and now I'm kind of caught in this weird space. It's almost like withdrawal from a chrome exploit 😐

how did you make it immediately unlock?

**Reply by Mazedotexe** [reply]
2B/2C immediately unlock, one of the best parts of Skiovox imo

**Reply by akabutnicer** [reply]
For real, that was one of the best findings I could have ever discovered. Immediate unlocking took time and work to discover.

**Reply by cloneyfirey** [reply]
i have 2a but it started doing it too :)
